roadmap.md

```diff
@@ -154,6 +154,14 @@ the catalogs ([FLINK-10232](https://issues.apache.org/jira/browse/FLINK-10232)).
 There is a broad effort to integrate Flink with the Hive Ecosystem, including
 metastore and Hive UDF support [FLINK-10556](https://issues.apache.org/jira/browse/FLINK-10556).
 
+There also a big effort to support Python for Table API [FLIP-38](https://cwiki.apache.org/confluence/display/FLINK/FLIP-38%3A+Python+Table+API).
```

Comment:
There 'is' also ...

Reply:
Must we add `is`, just double check?

Reply:
No offense. This is just a minor grammar thing that I thought you made a typo :)  'there' used like this case is always followed by an auxiliary verb, like "there is..., there are..., there will be..." etc